### PR TITLE
Santa Cruz eats

### PR DESCRIPTION
**Pull request by delaneyparker**

### CODE REVIEW

File: sunnyvale-eats.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Places to eat in Sunnyvale, CA
+
+- Dish N Dash
+- Hobee's
+- McDonalds
```
Comment (johnboy55) [inline sunnyvale-eats.md]: s/McDonalds/Burger King/

Comment (ebuceci) [inline sunnyvale-eats.md]: s/McDonalds|(Burger King)/In-N-Out/g
;)

Comment (delaneyparker) [inline sunnyvale-eats.md]: Maybe Kal’s Bar-B-Q?

**Comment by rgarapat** [comment]
@delaneyparker Have you eaten Indian food?

**Comment by ntend** [comment]
@delaneyparker How about the following?

```
El Palomar
Surfrider
```

**Comment by bryanjenningz** [comment]
@delaneyparker Vasili's Greek Restaurant

**Pull request merged**